broadleaf-thymeleaf3-presentation/src/main/java/org/broadleafcommerce/presentation/thymeleaf3/config/AbstractThymeleaf3DialectConfig.java

```diff
@@ -80,7 +80,7 @@ public Set<IProcessor> blDialectProcessors() {
     }
 
     @Configuration
-    @ConditionalOnClass(name = "nz.net.ultraq.thymeleaf.LayoutDialect")
+    @ConditionalOnClass(name = "nz.net.ultraq.thymeleaf.layoutdialect.LayoutDialect")
```

Comment:
I would think it is better to use value field of ConditionalOnClass annotation rather than name - in this case if in the future class/package will change again build will fail as import will reference a non existing class